### PR TITLE
Remove basic auth

### PR DESCRIPTION
**Pull request by theseanything**

### CODE REVIEW

**Review by brucebolt — APPROVED** [review]

**Comment by bilbof** [comment]
This'll remove the auth from staging too, right? Maybe we should set a flag in the deploy with `cf` to override this instead.

**Comment by paulmsmith** [comment]
Interested spectator 👋  - Wishing you all the best. I believe you can create 'draft' pull requests in Github these days so you don't have to worry about accidental untimely merging. You have control of changing it from draft to ready. 

image

**Comment by issyl0** [comment]
@paulmsmith Thanks for the head's up and the good wishes! We're working flat out here!

**Comment by paulmsmith** [comment]
I can see @nickcolley and all of you have been busy over the weekend. I'll keep an eye on your issues, if I can do quick grunt work and raise pull requests I will.

**Comment by NickColley** [comment]
@paulmsmith thanks for the offer mate but we're good for now, thank you again for thinking about us. :+1:

**Comment by paulmsmith** [comment]
@nickcolley No worries. More hands does add complication too. Ping me if there is anything.

**Review by richardTowers — CHANGES_REQUESTED** [review]
I think we need to remove:

```
    REQUIRE_BASIC_AUTH: true
```

From manifest.yml too

File: concourse/tasks/deploy-to-govuk-paas.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@ run:
       cf scale -i "$INSTANCES" govuk-coronavirus-vulnerable-people-form
 
       cf set-env govuk-coronavirus-vulnerable-people-form AWS_DYNAMODB_SUBMISSIONS_TABLE_NAME "$AWS_DYNAMODB_SUBMISSIONS_TABLE_NAME"
+      if [[ "${BASIC_AUTH_PASSWORD:-}" = "true" ]]; then
```
Comment (richardTowers) [inline concourse/tasks/deploy-to-govuk-paas.yml]:
```suggestion
      if [[ "${REQUIRE_BASIC_AUTH:-}" = "true" ]]; then
```

**Review by richardTowers — APPROVED** [review]

**Comment by sarahseewhy** [comment]
NOTE
@richardTowers and I un-paused the pipeline to deploy the application in production before merging this pull request.

This caused a basic-auth pop-up to appear for users for a 30 second period while we fixed the issue.

IMPACT
- request affected: ~2,000
- duration of incident: ~30 seconds
- time to resolution: ~30 seconds

Once Rich and I realised the mistake we merged the PR.

**Pull request merged**